pepper-apis/src/main/java/org/broadinstitute/ddp/service/KitCheckService.java

```diff
@@ -310,7 +311,7 @@ private void handleNextKit(Handle apisHandle, KitCheckResult kitCheckResult,
         if (schedule.getIndividualOptOutExpr() != null) {
             // Schedule allows opting out of individual kits, let's check it.
             try {
-                boolean shouldOptOut = interpreter.eval(schedule.getIndividualOptOutExpr(), apisHandle, userGuid, null);
+                boolean shouldOptOut = interpreter.eval(schedule.getIndividualOptOutExpr(), apisHandle, userGuid, userGuid, null);
```

Comment:
Let's `null` out the two call sites here as well (one here and one above).

Reply:
Nulled as well